### PR TITLE
Mass-Publish-Core

### PR DESCRIPTION
**Pull request by bradly-shorthand**
First upload of Mass-Publish working.

### CODE REVIEW

**Comment by ckortekaas** [comment]
Reading through this now. One minor thing that initially jumps out is that we should setup PHP Code Sniffer and some sort of autoformatter (like what Prettier does for JS etc.. and maybe even this too) following the WP coding formatting standards on https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/. Not required in this PR but something we should probably do at some point soon.

**Comment by ckortekaas** [comment]
@bradly-shorthand Could you please record a short video (eg using Zoom) of how this looks and works? Looks good.

**Comment by bradly-shorthand** [comment]
> Reading through this now. One minor thing that initially jumps out is that we should setup PHP Code Sniffer and some sort of autoformatter (like what Prettier does for JS etc.. and maybe even this too) following the WP coding formatting standards on https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/. Not required in this PR but something we should probably do at some point soon.

I agree, I spoke to James about getting the plugin cleaned up and we discussed getting this out first and then working towards tidying up the codebase as well as writing internal documentation on how it functions.

**Comment by bradly-shorthand** [comment]
> @bradly-shorthand Could you please record a short video (eg using Zoom) of how this looks and works? Looks good.

Video is being made, all clips recorded just an editing job right now.

File: src/shorthand_connect.php [hunk 1]
```diff
@@ -618,4 +619,42 @@ function shand_wpt_shorthand_extra_html()
 	echo '<textarea id="codearea" name="extra_html">' . esc_textarea($extra_html) . '</textarea>';
 }
 
-?>
+/* Brad Code */
```
Comment (james-shorthand) [inline src/shorthand_connect.php:622]: This can probably be removed

**Review by james-shorthand — APPROVED** [review]
It works as intended, I tested back to version 4 no issues arose.

**Review by ckortekaas — APPROVED** [review]

**Pull request merged**